src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -25,11 +25,11 @@
     {% if prepend or append %}
         <div class="input-group">
             {% if prepend %}
-                <span class="input-group-addon">{{ money_pattern|replace({ '{{ widget }}':''}) }}</span>
+                <span class="input-group-addon">{{ money_pattern|form_encode_currency|replace({ '{{ widget }}':''})|raw }}</span>
```

Comment:
the replacement should be done first, which will avoid the need for the `|raw` filter.